### PR TITLE
[16.0][FIX] base_report_to_printer: Expected singleton

### PR DESCRIPTION
**Pull request by willer-mo**
Fixes https://github.com/OCA/report-print-send/issues/314

### CODE REVIEW

File: base_report_to_printer/models/ir_actions_report.py [hunk 1]
```diff
@@ -159,14 +159,14 @@ def _render_qweb_pdf(self, report_ref, res_ids=None, data=None):
         document, doc_format = super()._render_qweb_pdf(
             report_ref=report_ref, res_ids=res_ids, data=data
```
Comment (alexis-via) [inline base_report_to_printer/models/ir_actions_report.py]: As report_ref is not a named argument, I would also remove "report_ref=" in the super()

**Review by alexis-via — APPROVED** [review]

**Comment by alexis-via** [comment]
Thanks for this bugfix PR. Obviously, the people work ported base_report_to_printer didn't do their work seriously and didn't even bother to test their code. Thanks for repairing all this!

File: base_report_to_printer/models/ir_actions_report.py (hunk 1 above)
Comment (willer-mo) [reply base_report_to_printer/models/ir_actions_report.py]: Done :+1:

**Comment by alexis-via** [comment]
We need a maintainer to validate this PR rapidly ; this bug is critical because it make all qweb-pdf report fail !

**Comment by willer-mo** [comment]
@sebalix

**Pull request merged**